python/hail/tests/test_methods.py

```diff
@@ -1608,26 +1608,34 @@ def test_ld_prune(self):
                     (filtered_ds['GT'].n_alt_alleles() - filtered_ds['mean'])
                     * filtered_ds['sd_reciprocal'] * (1 / hl.sqrt(n_samples)), 0))
 
-        block_matrix = BlockMatrix.from_entry_expr(normalized_mean_imputed_genotype_expr)
-        entries = ((block_matrix @ block_matrix.T) ** 2).entries()
+        std_bm = BlockMatrix.from_entry_expr(normalized_mean_imputed_genotype_expr)
+
+        self.assertEqual(std_bm.n_rows, 14)
+
+        entries = ((std_bm @ std_bm.T) ** 2).entries()
 
         index_table = filtered_ds.add_row_index().rows().key_by('row_idx').select('locus')
         entries = entries.annotate(locus_i=index_table[entries.i].locus, locus_j=index_table[entries.j].locus)
 
-        contig_filter = entries.locus_i.contig == entries.locus_j.contig
-        window_filter = (hl.abs(entries.locus_i.position - entries.locus_j.position)) <= 1000000
-        identical_filter = entries.i != entries.j
+        bad_pair = (
+            (entries.entry >= 0.1) &
```

Comment:
This R2 threshold is different than defined above. Was that intentional?

Reply:
good catch, fixing. It makes sense that the test still passed, because 0.1 is less stringent.